dwave/cloud/client.py

```diff
@@ -147,6 +155,8 @@ class Client(object):
 
     # Number of worker threads for each problem processing task
     _SUBMISSION_THREAD_COUNT = 5
+    _UPLOAD_PROBLEM_THREAD_COUNT = 1
+    _UPLOAD_PART_THREAD_COUNT = 10
```

Comment:
Out of curiosity, where do these numbers come from?

Reply:
Just initial estimates. Actual workers are extremely IO-bound, so we might as well just resort to [the default](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor) in the standard library (which varies across versions; and in Py3 actual number of workers is adaptive but less then the given max).